src/solutions/auth/auth.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
similarly for consistency, can we can rename all `.js` files to `.ts`?
We don't have to do strict typing, but it would make sense in the rest of the project.